src/libs/actions/Task.js

```diff
@@ -260,9 +266,25 @@ function completeTask(taskReportID, taskTitle) {
     ];
 
     // Update optimistic data for parent report action
-    const optimisticParentReportData = ReportUtils.getOptimisticDataForParentReportAction(taskReportID, completedTaskReportAction.created, CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD);
-    if (!_.isEmpty(optimisticParentReportData)) {
-        optimisticData.push(optimisticParentReportData);
+    const optimisticDataForParentReportAction = ReportUtils.getOptimisticDataForParentReportAction(taskReportID, completedTaskReportAction.created, CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD);
```

Comment:
Let's say we have a report with ID `1` and it has a created action and another action for a task and the reportActionID for these are `11` and `12` respectively. So this optimistic data would correspond to the `12` reportActionID, right?

Reply:
Suppose that you logged in with userA. When we create a task with assignee userB and shareDestination groupC, it would create a taskReport(`1`) and create two report actions in assignee report(chat with userB) and shareDestination report(group chat) respectively. Assuming reportActionID for reportAction in group chat is `11` and reportActionID for action in assignee report is `12`.

This code would get optimistic data for reportAction `11`

Reply:
Sorry it's not clear what does this action corresponds to 
```
Assuming reportActionID for reportAction in group chat is 11 and reportActionID for action in assignee report is 12.`
```

Reply:
> We create 2 actions when assignee and shareDestination is different, one in assignee report, one in share destination

Aaah, I see, that was the confusing part for me. Makese sense now.